source/blog/2017-11-20-ember-2-17-released.md

```diff
@@ -0,0 +1,149 @@
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 2.17
+
+Ember.js 2.17 is an incremental, backwards compatible release of Ember with
+bugfixes, performance improvements, and minor deprecations.
+
+#### Deprecations in Ember 2.17
+
+Deprecations are added to Ember.js when an API will be removed at a later date.
+
+Each deprecation has an entry in the deprecation guide describing the migration
+path to more stable API. Deprecated public APIs are not removed until a major
+release of the framework.
+
+Consider using the
+[ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow)
+addon if you would like to upgrade your application without immediately addressing
+deprecations.
+
+Two new deprecations are introduces in Ember.js 2.17:
+
+* TODO
+* TODO
+
+For more details on changes in Ember.js 2.17, please review the
+[Ember.js 2.17.0 release page](https://github.com/emberjs/ember.js/releases/tag/v2.17.0).
+
+### Upcoming Changes in Ember.js 2.18
+
+Ember.js 2.18 will introduce two new features:
+
+* TODO
+* TODO
+
+#### Deprecations in Ember.js 2.18
+
+Two new deprecations are introduces in Ember.js 2.18:
+
+* TODO
+* TODO
+
+For more details on the upcoming changes in Ember.js 2.18, please review the
+[Ember.js 2.18.0-beta.1 release page](https://github.com/emberjs/ember.js/releases/tag/v2.18.0-beta.1).
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 2.17
+
+#### Deprecations in Ember Data 2.17
+
+Two new deprecations are introduces in Ember Data 2.17:
+
+* TODO
+* TODO
+
+For more details on changes in Ember Data 2.17, please review the
+[Ember Data 2.17.0 release page](https://github.com/emberjs/data/releases/tag/v2.17.0).
+
+
+### Upcoming changes in Ember Data 2.18
+
+
+#### Deprecations in Ember Data 2.18
+
+For more details on the upcoming changes in Ember Data 2.18, please review the
+[Ember Data 2.18.0-beta.1 release page](https://github.com/emberjs/data/releases/tag/v2.18.0-beta.1).
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js
+applications.
+
+### Upgrading Ember CLI
+
+You may upgrade Ember CLI separately from Ember.js and Ember Data! To upgrade
+your projects using `yarn` run:
+
+```
+yarn upgrade ember-cli
+```
+
+To upgrade your projects using `npm` run:
+
+```
+npm install --save-dev ember-cli
+```
+
+After running the
+upgrade command run `ember init` inside of the project directory to apply the
+blueprint changes. You can preview those changes for [applications](https://github.com/ember-cli/ember-new-output/compare/v2.15.0...v2.17.0)
+and [addons](https://github.com/ember-cli/ember-addon-output/compare/v2.15.0...v2.17.0).
+
+### Changes in Ember CLI 2.17
```

Comment:
- [#7317](https://github.com/ember-cli/ember-cli/pull/7317) Introduce broccoli-assembler 

internal refactorings of the build pipeline to make it more flexible and composable

- [#7342](https://github.com/ember-cli/ember-cli/pull/7342) Use "ci" mode for testem.js 

by default `ember test` will now run headless chrome, while `ember test --server` opens a Chrome window

- [#7353](https://github.com/ember-cli/ember-cli/pull/7353) remove ember-cli-shim warning

Ember CLI previously displayed a warning when you removed `ember-cli-shims`, but the dependency is no longer needed if none of your dependencies are using the old shims and all of them are using `ember-cli-babel` 6.6.0 or above.

- [#7369](https://github.com/ember-cli/ember-cli/pull/7369) Fix issue with linting within an addon without an app/ directory.
- [#7378](https://github.com/ember-cli/ember-cli/pull/7378) Ensure test-support and addon-test-support are linted.

- [#7416](https://github.com/ember-cli/ember-cli/pull/7416) Add support for Node 9.
- [#7417](https://github.com/ember-cli/ember-cli/pull/7417) Issue warning for Node 7.

- [#7436](https://github.com/ember-cli/ember-cli/pull/7436) ember-try: retire 2.8 scenario, introduce 2.16